tests/nativeapp/patch_utils.py

```diff
@@ -0,0 +1,18 @@
+from unittest import mock
```

Comment:
Can ignore all test changes for now, those should be resolved once the upstream branch's PR has been merged in.
This comment will be resolved once that happens, and new tests are available for review.